CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.1.0]
+### Changed
+- Update InfuraJsonRpcSupportedNetwork with supported networks ([#69](https://github.com/MetaMask/eth-json-rpc-infura/pull/69))
```

Comment:
Typically when we have a change that renames or removes types we mark it as a breaking change. Here we aren't doing that, but we _are_ removing strings that the type previously accepted. Would this count as a breaking change? Granted, it wouldn't have any runtime effects, just compile-time effects.

Reply:
Yeah good point, it probably should be v8 instead